### PR TITLE
Add White arrows. Video Poker icon.

### PR DESCRIPTION
**Pull request by Rdzanuwu**

### CODE REVIEW

File: HTML/White Arrows.html [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+                        <th>▽</th>
+                        <th>▷</th>
+                        <th>▽</th>
+                        <th>◁</th>
+                        <th>◁</th>
+                    </tr>
+                </table>                
+        </div>
+        <div class="page-footer relative-footer">Page 3 of 3</div>
+    </div>
```
Comment (MrPeanut1028) [inline HTML/White Arrows.html:471]: Spacing is off starting on the last page, I think you might also be missing a `<div>` in the end of the document. Line everything up and it'll be easy to see.

**Review by MrPeanut1028 — CHANGES_REQUESTED** [review]

File: HTML/White Arrows.html (hunk 1 above)
Comment (Rdzanuwu) [reply HTML/White Arrows.html:469]: Not sure if <div> with section needs closing tag because i looked in the other manual and it doesnt have one.

Comment (kavinkul) [reply HTML/White Arrows.html:470]: All div ending tags are required according to w3. If you see them missing, that usually are mistakes which I have seen in some manuals in the past.

Comment (Rdzanuwu) [reply HTML/White Arrows.html:470]: Yeah in 0 manual div section is missing a closing tag

**Pull request merged**